tensorflow/core/kernels/mkl_conv_ops.h

```diff
@@ -413,25 +422,11 @@ class MklDnnConvUtil {
     }
 
     if (is_conv2d) {
-      // Conv + pad fusion is enabled only for 2D
+      // Conv + pad fusion is enabled only for 2D.
       // If pad_enabled, i.e., pad and conv op are fused, then
       // all pads are already passed from pad op through
-      // *pad_l and *pad_r
-      if (pad_enabled) {
-        pad_top = static_cast<int64>((*pad_l)[0]);
-        pad_left = static_cast<int64>((*pad_l)[1]);
-        pad_bottom = static_cast<int64>((*pad_r)[0]);
-        pad_right = static_cast<int64>((*pad_r)[1]);
-        // update the out_rows and out_cols based on all
-        // sides of the pads coming from pad op.
-        out_rows = out_rows + (pad_top + pad_bottom) / stride_rows;
```

Comment:
Why don't we need to update `out_rows` and `out_cols` anymore?

Reply:
Because previously, calling GetWindowedOutputSizeVerboseV2() was handling only subset of the padding cases to compute out_rows and out_cols. With this change, GetWindowedOutputSizeVerboseV2() will handle all cases of padding.

Reply:
Thank you for the explanation!